### PR TITLE
CODENVY-703; do not show terminal by default in factories

### PR DESCRIPTION
**Pull request by mshaposhnik**
@vparfonov @vitaliy-guliy

### CODE REVIEW

**Comment by vitaliy-guliy** [comment]
Great!

**Comment by riuvshin** [comment]
Super! Thanks Maxim!

File: plugins/plugin-machine/che-plugin-machine-ext-client/src/main/java/org/eclipse/che/ide/extension/machine/client/MachineExtension.java [hunk 1]
```diff
@@ -94,15 +94,6 @@ public MachineExtension(MachineResources machineResources,
             @Override
             public void onWsAgentStarted(WsAgentStateEvent event) {
                 machinePortProvider.get();
```
Comment (sleshchenko) [inline plugins/plugin-machine/che-plugin-machine-ext-client/src/main/java/org/eclipse/che/ide/extension/machine/client/MachineExtension.java:96]: Do you know why we need invoke provider here?

Comment (mshaposhnik) [inline plugins/plugin-machine/che-plugin-machine-ext-client/src/main/java/org/eclipse/che/ide/extension/machine/client/MachineExtension.java:96]: No (

Comment (sleshchenko) [inline plugins/plugin-machine/che-plugin-machine-ext-client/src/main/java/org/eclipse/che/ide/extension/machine/client/MachineExtension.java:96]: Looks like we can remove it. @vparfonov WDYT?

Comment (vparfonov) [inline plugins/plugin-machine/che-plugin-machine-ext-client/src/main/java/org/eclipse/che/ide/extension/machine/client/MachineExtension.java:96]: Yep, you are right. Looks like we can remove it.

**Comment by vparfonov** [comment]
ok

**Pull request merged**